connections/management/commands/empty_mapped_apartments_database.py

```diff
@@ -0,0 +1,18 @@
+import logging
+from django.core.management.base import BaseCommand
+
+from connections.models import MappedApartment
+
+_logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = "Empty MappedApartment database table"
+
+    def handle(self, *args, **options):
+        try:
+            MappedApartment.objects.all().delete()
+            _logger.info("Succefully deleted mapped apartments from database")
+        except Exception as e:
+            _logger.error("Could not delete mapped apartments from database:", str(e))
```

Comment:
I think a bit more Pythonic would be to use `_logger.exception("Could not delete mapped apartments from database:")`. It still uses `ERROR` log level, but will log the current exception (`e`) automatically after printing the message.